src/Symfony/Component/Console/Command/Command.php

```diff
@@ -572,6 +572,8 @@ public function getSynopsis($short = false)
      * Add a command usage example.
      *
      * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return $this
```

Comment:
we are using `@return self` everywhere else. Not sure which one is the right one, but I didn't find any occurrence of `@return $this` in Symfony codebase.